config.sh

```diff
@@ -7,44 +7,39 @@ function build_wheel {
     local lib_plat=$PLAT
     if [ -n "$IS_OSX" ]; then
         install_gfortran
+    else
+        # For manylinux2010 builds with manylinux1 openblas builds
+        $use_sudo yum install -y libgfortran-4.4.7
     fi
     build_libs $lib_plat
     # Fix version error for development wheels by using bdist_wheel
     build_bdist_wheel $@
 }
 
 function build_libs {
-    local plat=${1:-$PLAT}
-    # Force 64-bit OpenBLAS library for macOS intel (dual arch)
-    # builds. For these builds, we pretend to be dual arch, but in
-    # fact we're only using the 64-bit build of OpenBLAS
-    if [ -n $IS_OSX ] && [ $plat == intel ]; then
-        plat=x86_64
-    fi
-    local tar_fname=$(get_gf_lib "openblas-${OPENBLAS_VERSION}" "$plat")
-    local tar_path=$(abspath $tar_fname)
-    # Sudo needed for macOS
-    local use_sudo=""
-    [ -n "$IS_OSX" ] && use_sudo="sudo"
-    (cd / && $use_sudo tar zxf $tar_path)
+    # Use the same incantation as numpy/tools/travis-before-install.sh
+    python -mpip install urllib3
+    target=$(python numpy/tools/openblas_support.py)
```

Comment:
Strange to call a command a target. Maybe explicitly use `python` on the next line. Where does the file come from?

Reply:
This is a strange construction, it could use a comment.

Reply:
Looks like what it does is download all the libraries into `/var/tmp/openblas/`. This could become rather inefficient if the libraries become numerous.

Reply:
Adding a comment and a todo to use site.cfg instead of copying.